fern/docs-config-v2/definition/docs.yml

```diff
@@ -0,0 +1,310 @@
+
+  TabbedNavigationConfig:
+    type: list<TabbedNavigationItem>
+
+  TabbedNavigationItem:
+    properties:
+      tab: TabId
+      layout: list<NavigationItem>
+
+  NavigationItem:
+    discriminated: false
+    union:
+      - PageConfiguration
+      - SectionConfiguration
+      - ApiSectionConfiguration
+
+  LogoConfiguration:
+    properties:
+      image-path: optional<ImageConfig>
+      href:
+        type: optional<string>
+        docs: If defined, the logo will be wrapped in an anchor tag with this href
+
+  DocsTypographyConfig:
+    properties:
+      headings: optional<FontConfig>
+      body: optional<FontConfig>
+      code: optional<FontConfig>
+
+  FontConfig:
+    properties:
+      font-path:
+        type: string
+        docs: The relative path of the font file.
+      name:
+        type: optional<string>
+        docs: |
+          If the `name` is not supplied, Fern will default it to a generated name that will be used to reference your custom font in the eventually injected CSS in the docs.
+
+  PageConfiguration:
+    properties:
+      page: string
+      path: string
+      slug: optional<string>
+
+  SectionConfiguration:
+    properties:
+      section: string
+      contents: list<NavigationItem>
+      collapsed: optional<boolean>
+      slug: optional<string>
+
+  ApiSectionConfiguration:
+    properties:
+      api: string
+      api-name:
+        type: optional<string>
+        docs: Name of API that we are referencing
+      audiences: optional<list<string>>
+      display-errors:
+        type: optional<boolean>
+        docs: Defaults to false
+      snippets: optional<SnippetsConfiguration>
+
+  SnippetsConfiguration:
+    properties:
+      python: optional<string>
+      typescript: optional<string>
+      go: optional<string>
+      java: optional<string>
+
+  DocsColorsConfig:
+    properties:
+      accent-primary: optional<ColorConfig>
+      background: optional<ColorConfig>
+      foreground:
+        type: optional<ColorConfig>
+        docs: Defaults to transparent
+      snippets-background:
+        type: optional<ColorConfig>
+        docs: Defaults to colors.foreground
+      border:
+        type: optional<ColorConfig>
+        docs: Defaults to white/10 in dark mode, and black/10 in light mode
+
+  ColorConfig:
+    properties:
+      dark: optional<string>
+      light: optional<string>
+
+  ImageConfig:
+    properties:
+      dark: optional<string>
+      light: optional<string>
+    docs: Path to the image.
+
+  HeaderConfig:
+    properties:
+      position:
+        type: optional<HeaderPosition>
+        docs: Defaults to "sticky"
+      render-mode: optional<HeaderRenderMode>
+      header-height-rem:
+        type: optional<double>
+        docs: Defaults to 4rem (64px)
+      border-radius-rem:
+        type: optional<double>
+        docs: If set, the header will have rounded corners, and a border around the entire header.
+      links: optional<list<HeaderLink>>
```

Comment:
I think we should try to write out this config for cohere -- feels like it can get very nested
```yaml
header: 
  position: 
    type: sticky
    value: 
      type: solid
      background-color:  
        dark: #e....
        light: #e....
```

Reply:
Thoughts on something more like: 

```yaml
header: 
  position: sticky
  backdrop: 
    type: blur
    rem:  # blur-rem
    color:  # blur-color
```

```yaml
header: 
  position: static
  backdrop: 
    type: blur # <------ throw an error
```

```yaml
header: 
  position: static
  backdrop: 
    type: solid
     color: # <------ background-color
```

Because this is the `authoring format` it actually pays to be a little less type safe. We read the config, validate it, and turn it into a more type safe config at read time.

Reply:
Another option: 

```yaml
header: 
  position: sticky
  backdrop-blur: 
    color: ...
    rem: ...
```

```yaml
header: 
  position: sticky
  backdrop-solid: 
    color: ...
    rem: ...
```